canonical-web-and-design/ubuntu.com
# Build /managed/cassandra

## Done

- Build new `/managed/cassandra` page
- Add to secondary nav 
- Add contact form at `shared/forms/interactive/managed-cassandra`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed/cassandra
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1iYMH1ISY4uDtr3ZzH0f_hnEzvqaeufO43z__cMxs0vQ/edit?ts=5f71f2cb#heading=h.xj070le8yswb)
- Check modal form against [form doc](https://docs.google.com/document/d/11QCRbr4aqRGEYXhU-9ob3Y4tYYLhrubJWimiaYFRVco/edit#)


## Issue / Card

Fixes [#3360](https://github.com/canonical-web-and-design/web-squad/issues/3360)
